### ISSUE TITLE
Cannot seek videos on rule34.paheal.net

### ISSUE THREAD

**Issue by xxxwangfirexxx** [shish/shimmie2#893]
I cannot seek through videos on rule34.paheal.net except with Firefox, no matter what device I use. I'll use https://rule34.paheal.net/post/view/4848810#search=animated%20grandcupido as an example, in Google Chrome if I try to seek forward by clicking on the timeline nothing happens and the videos just keeps playing where its at, but if I do the same thing in Firefox it will actually jump to where I select.

**Comment by tegaki-tegaki** [comment]
This seems to be yet another case of browser 'quirkiness', so not really an issue related to shimmie / paheal (does paheal get fixes through the shimmie project? I guess paheal is "just an extension + a theme" on shimmie 🤔 )

see: https://stackoverflow.com/questions/35751736/seeking-in-html5-video-with-chrome

I confirmed that it does indeed behave like this in chrome (yet works in FF)... really interesting

tl;dr: Chrome loves to over-optimize (and have weird "bugs") and FF tends to do things "according to spec with zero surprises".

**Comment by shish** [comment]
This also seems to be video-specific; like some videos can seek in ff+chrome, while others only seek in ff...

**Comment by tegaki-tegaki** [comment]
> This also seems to be video-specific; like some videos can seek in ff+chrome, while others only seek in ff...

yeah, in the stackoverflow the person talks about this too, chrome does some filesize checks, if it's small enough, then chrome will download the entire file at once and enable seeking. But if it's big enough, it'll download a chunk at a time, but in the process somehow break file seeking. (I think video codecs and seeking can be a "can of worms" in its own right, and especially so if you don't have the entire file available)
